aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1359,9 +1361,8 @@ module aptos_framework::stake {
         assert!(exists<OwnerCapability>(owner), error::not_found(EOWNER_CAP_NOT_FOUND));
     }
 
-    // Will be deleted after transaction_fee has its own MintCap for storage refunds.
-    public(friend) fun copy_aptos_coin_mint_cap_for_storage_refund(): MintCapability<AptosCoin> acquires AptosCoinCapabilities {
```

Comment:
do we want to keep acquire to keep the function same as before

Reply:
You can just delete this function. public(friend) is treated the same as private so it's good to delete